ui/imports/shared/popups/send/controls/GasSelector.qml

```diff
@@ -16,18 +16,19 @@ Item {
 
     property var selectedAsset
```

Comment:
The only thing taken from that object is `symbol`. It should be taken just as `symbol` property.